TrafficCapture/dockerSolution/src/main/docker/migrationConsole/workflows/templates/createSnapshot.yaml

```diff
@@ -21,7 +22,7 @@ spec:
             value: ""
             description: "Which indices should be snapshot.  Default empty.  Empty string means all."
           - name: snapshot-config
-            value: "{ \"snapshot_name\": \"{{inputs.parameters.snapshot-name}}\", \"s3\": {{inputs.parameters.s3-config}} }"
+            value: "{ \"snapshot_name\": \"{{inputs.parameters.snapshot-name}}\", \"otel_endpoint\": \"{{inputs.parameters.otel-collector-endpoint}}\" , \"s3\": {{inputs.parameters.s3-config}} }"
```

Comment:
regardless of anything else, why would this be part of snapshot-config?  IIRC, this value is passed along later on to RFS.  Why would RFS want to know the otel-endpoint?  Again, parameter passing is changing considerably, so I don't think that any changes will be necessary because we'll refine it in https://github.com/opensearch-project/opensearch-migrations/pull/1633.